code/controlconfig/controlsconfig.cpp

```diff
@@ -516,7 +516,9 @@ void control_config_list_prepare()
 	Num_cc_lines = y = z = 0;
 	while (z < CCFG_MAX) {
 		if (Control_config[z].tab == Tab && !Control_config[z].disabled) {
-			if (Control_config[z].hasXSTR) {
+			if (Control_config[z].hasXSTR > 1) {
+				Cc_lines[Num_cc_lines].label = XSTR(Control_config[z].text, Control_config[z].hasXSTR);
```

Comment:
I don't like using `XSTR` with a non constant parameter set since it makes it impossible to automatically generate `strings.tbl`. Not sure how to fix that though since I don't think that the text comes from a table.

Reply:
Just changed the name. I see what you mean about not being able to automatically generate strings.tbl. Yeah I couldn't find many entries in `strings.tbl` that were actually listed in the code base.

Reply:
If this would be a critical issue I could make a new PR which just assigns the controls that don't have a unique strings.tbl index to have a unique index at 1570 or above?

Reply:
No, this is not critical. We need to fix this at some point anyway but that out of scope for this.

Reply:
Gotcha. At the very least this PR allows for all the controls to be translatable via `strings.tbl` now.